cmd/root.go

```diff
@@ -0,0 +1,50 @@
+package cmd
+
+import (
+	"os"
+
+	log "github.com/sirupsen/logrus"
+
+	"github.com/spf13/cobra"
+)
+
+var Debug bool
+var Quiet bool
+var PrintLevel string
+var SkipDirList []string
+
+func init() {
+	RootCmd.AddCommand(CmdAnsible)
+
+	RootCmd.PersistentFlags().BoolVarP(&Debug, "debug", "d", false, "debug level logging")
+	RootCmd.PersistentFlags().BoolVarP(&Quiet, "quiet", "q", false, "run with error level only logging")
```

Comment:
Make it consistent to the description above.

```suggestion
	RootCmd.PersistentFlags().BoolVarP(&Quiet, "quiet", "q", false, "error level logging")
```